### ISSUE TITLE
The problem of data partitioning

### ISSUE THREAD

**Issue by tangmimimimi** [wangyiting07/CANDA#1]
Based on your publicly available data_utils.py code on GitHub, I extracted data from the SEED-IV original dataset into the following structure. The directory structure for eye features is identical.

Dataset
├── Subject1
│ ├── 1
│    ├── eeg_de_1.npy, ..., eeg_de_4.npy
│ ├── 2
│ └── 3
├── Subject2
│ ├── 1
│ ├── 2
│ └── 3
├── ...

I trained the model using the command:

bash

python train_seed.py --dataset seediv --lr 1e-4 --wd 1e-3 --alpha 0.8 --epochs 100 --num_heads 3 --common_dim 126 --num_layers 3

During debugging, I noticed that the shape of the intermediate training data is (16, 5, 310), while the test data shape is (7, 5, 31). The accuracy (acc) during training remains consistently low. Could this be due to the data splitting method? If possible, could you share how you split the data during your training?

Additionally, my training completed in a relatively short time, which differs significantly from the two weeks mentioned in your paper. Could this also be related to the data splitting approach?

**Comment by wangyiting07** [comment]
Thanks for your comment! Since we have communicated through email, I will close this comment and we can keep the conversation going there.
